cli/args/flags.rs

```diff
@@ -1025,17 +1158,15 @@ fn clap_root() -> Command {
     .arg(
       Arg::new("unstable-bare-node-builtins")
         .long("unstable-bare-node-builtins")
-        .help("Enable unstable bare node builtins feature")
-        .env("DENO_UNSTABLE_BARE_NODE_BUILTINS")
+        .help("Enable unstable bare node builtins feature (or set DENO_UNSTABLE_BARE_NODE_BUILTINS=true).")
         .value_parser(FalseyValueParser::new())
         .action(ArgAction::SetTrue)
         .global(true),
     )
     .arg(
       Arg::new("unstable-byonm")
         .long("unstable-byonm")
-        .help("Enable unstable 'bring your own node_modules' feature")
-        .env("DENO_UNSTABLE_BYONM")
```

Comment:
Not including this here means it doesn't show in --help anymore

Reply:
They need to manually added to `ENV_VARIABLES_HELP`